src/IO/ProcessRunner.php

```diff
@@ -13,4 +13,6 @@
      * @return Process
      */
     public function run(Process $process, $mustSucceed = true);
+
+    public function setUserInteraction(UserInteraction $userInteraction);
```

Comment:
Actually, we can have the "user interaction" in the constructor of these class, but we are just moving the problem into this class: we need, for instance at the [`ConsoleEvents::COMMAND` event](http://symfony.com/doc/current/components/console/events.html#the-consoleevents-command-event) to call a `registerOutput` or `setOutput` or `attachOutput` to the UserInteraction class.